src/ray/common/id.cc

```diff
@@ -155,6 +145,27 @@ const ActorHandleID ComputeNextActorHandleId(const ActorHandleID &actor_handle_i
   return ActorHandleID::FromBinary(std::string(buff, buff + ActorHandleID::Size()));
 }
 
+JobID JobID::FromDriverId(const WorkerID &driver_id) {
```

Comment:
I think it'd more natural to do `driver_id.GetJobId`, instead of `JobID::FromDriverId(driver_id)`

Reply:
agree with you, but currently `WorkerID` is `UniqueID`,  we're not able to add `GetJobId ` for WorkerID until we separate `WorkerID` from `UniqueID`. But I think it's okay to leave a TODO to  note us this point.

Reply:
then let's use a helper function and put a TODO there?